src/migrations/20230912133202-update-class-goal.js

```diff
@@ -0,0 +1,43 @@
+const {
+  prepMigration,
+} = require('../lib/migration');
+
+const originalGoalName = '(PILOT) Grant recipient will improve teacher-child interactions (as measured by CLASS scores)';
+const updatedGoalName = '(Monitoring) Grant recipient will improve teacher-child interactions (as measured by CLASS scores)';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+
+      await queryInterface.sequelize.query(`
```

Comment:
There is one instance of the goal that does not have the goalTemplateId set.  Goal ID is 45715, goalTemplateId is 18172